addon/components/ember-attacher-inner.js

```diff
@@ -18,6 +18,8 @@ export default Component.extend({
   init() {
     this._super(...arguments);
 
+    this.set('_shouldRenderYieldBlock', !this.get('lazyRender'));
```

Comment:
`_shouldRenderYieldBlock` could be moved into a computed that depends on 'lazyRender'. Also, I think we should call it `_shouldRender` since it also takes care of rendering the arrow and circle (for the fill animation).

```javascript
_shouldRender: computed('lazyRender', function() {
  return !this.get('lazyRender');
})
```

Setting this property on our first show event should still work as expected.

Side note: As a matter of best practices, use `this.myProperty = something` in `init` since `this.set()` is used to notify ember of property changes. Since we're initializing the component, there is no need to notify of changes. Basically comes down to a small performance win by avoiding unesc work.